Source/WebGPU/WebGPU/RenderBundle.mm

```diff
@@ -30,15 +30,18 @@
 
 namespace WebGPU {
 
-RenderBundle::RenderBundle(id<MTLIndirectCommandBuffer> indirectCommandBuffer, Vector<BindableResource>&& resources, Device& device)
+RenderBundle::RenderBundle(id<MTLIndirectCommandBuffer> indirectCommandBuffer, const WGPURenderBundleEncoderDescriptor& descriptor, Vector<BindableResource>&& resources, RefPtr<PipelineLayout>&& pipelineLayout, Device& device)
     : m_indirectCommandBuffer(indirectCommandBuffer)
     , m_device(device)
+    , m_descriptor(descriptor)
     , m_resources(WTFMove(resources))
+    , m_pipelineLayout(WTFMove(pipelineLayout))
 {
 }
 
 RenderBundle::RenderBundle(Device& device)
     : m_device(device)
+    , m_descriptor()
```

Comment:
is this necessary?

Reply:
Because it is const. But I think I can initialize it in the header